packages/icons/src/index.ts

```diff
@@ -12,52 +12,66 @@ o888o o888o o888o o888o  888bod8P' `Y8bod8P' d888b      "888" 8""888P'
 
 // System Icons
 import { default as ArrowLeft } from "./components/ArrowLeft";
+import { default as ArrowRight } from "./components/ArrowRight";
+import { default as ArrowUp } from "./components/ArrowUp";
+import { default as ArrowDown } from "./components/ArrowDown";
 import { default as Backup } from "./components/Backup";
 import { default as Bell } from "./components/Bell";
-import { default as CancelCircleFilled } from "./components/CancelCircleFilled";
-import { default as CancelCircle } from "./components/CancelCircle";
+import { default as Billing } from "./components/Billing";
+import { default as Calendar } from "./components/Calendar";
 import { default as Cancel } from "./components/Cancel";
+import { default as CancelCircle } from "./components/CancelCircle";
+import { default as CancelCircleFilled } from "./components/CancelCircleFilled";
 import { default as CaretDown } from "./components/CaretDown";
 import { default as CaretFilledDown } from "./components/CaretFilledDown";
```

Comment:
one day we'll probably want to come back and make these `import ArrowLeft from "./components/ArrowLeft";` 😃 

Not in this PR though, there's already enough going on!

Reply:
What is the impact of that change?

Reply:
There's fewer characters per line, that's about it :) `import { default as Foo } from` and `import Foo from` are equivalent!